src/preprocessing/preprocessors.py

```diff
@@ -140,29 +141,33 @@ def _load(self):
         This is done not at __init__ time because some of this are dynamically
         generated files, so doesn't need to happen at import time.
         """
-        viparts = self._vip_articles = set()
+        viparts = self._vip_articles = dict()
 
         # some manually curated pages
         if config.DESTACADOS is not None:
             with open(config.DESTACADOS, 'rt', encoding='utf8') as fh:
-                for line in fh:
-                    viparts.add(line.strip())
+                viparts.update({to3dirs._quote(line.strip()): 1 for line in fh})
 
         # must include according to the config
-        viparts.update(config.langconf['include'])
+        viparts.update({include: 2 for include in config.langconf['include']})
 
         # portal articles from the front-page portal, itself included
-        viparts.add(config.langconf['portal_index'])
+        viparts.update({config.langconf['portal_index']: 2})
         _path = os.path.join(config.DIR_TEMP, 'portal_pages.txt')
         with open(_path, 'rt', encoding='utf-8') as fh:
-            viparts.update(line.strip() for line in fh)
+            viparts.update({to3dirs._quote(line.split()): 2 for line in fh})
+
+        # add test-infra articles if test mode is enable.
+        if config.TEST_MODE is not None:
+            with open('test_infra.txt', 'rt', encoding='utf8') as fh:
+                viparts.update({to3dirs._quote(line.split()[0]): 3 for line in fh})
 
         logger.info("Loaded %d VIP articles", len(viparts))
 
     def __call__(self, article):
         if self._vip_articles is None:
             self._load()
-        return article in self._vip_articles
+        return self._vip_articles[article] if article in self._vip_articles else None
```

Comment:
O simplemente `return self._vip_articles.get(article)`

Reply:
Más simple incluso: `return article in self._vip_articles`

Reply:
Me gustaba mas la opcion de @fzuccolo porque ya devolvia el multiplicador.